app/models/person.rb

```diff
@@ -83,7 +83,7 @@ def target_legislators(json: false, count: Ivr::Call::MAXIMUM_CONNECTIONS)
     remaining_count = count - locals.count
     others = other_targets(count: remaining_count, excluding: locals)
     if json
-      locals.as_json('local' => true) + others.as_json('local' => false)
+      locals.as_json(extras: { 'local' => true }) + others.as_json(extras: { 'local' => false })
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.